idseq_dag/engine/pipeline_flow.py

```diff
@@ -206,6 +211,17 @@ def write_invalid_input_json(self, error_json):
 
         idseq_dag.util.s3.upload_with_retries(local_input_errors_file, self.output_dir_s3 + "/")
 
+    def create_status_json_file(self):
+        ''' Create [stage name]_status.json, which will include step-level job status updates to be used in idseq-web. '''
+        log.write(f"Creating {self.name}_status.json")
+        local_status_file = f"{self.output_dir_local}/{self.name}_status.json"
```

Comment:
There should a unique point of code where the filename is created. 
You can either define it early as an instance variable or create a function that returns the file.